lib/puppet/version.rb

```diff
@@ -72,6 +72,10 @@ def self.version=(version)
     @puppet_version = version
   end
 
+  def self.major_minor_version
+    self.version.sub(/\A(\d+\.\d+)\..*$/, '\1')
+  end
```

Comment:
Can you add a unit test for this? Btw, there's a related PR in https://github.com/puppetlabs/puppet/pull/5720, with some discussion about using semantic puppet to parse the version string. Would be good to consolidate these PRs. /cc @binford2k @MosesMendoza